lib/kitchen/verifier/shell.rb

```diff
@@ -36,16 +36,23 @@ class Shell < Kitchen::Verifier::Base
       default_config :command, "true"
       default_config :shellout_opts, {}
       default_config :live_stream, $stdout
+      default_config :remote, false
```

Comment:
The cloud/hypervisor types are notions of ``driver`` , not ``remote`` imho

Example of existing method from ``lib/kitchen/instance.rb``:
```
    # Executes an arbitrary command on this instance.                           
    #                                                                           
    # @param command [String] a command string to execute                       
    def remote_exec(command)                                                    
      transport.connection(state_file.read) do |conn|                           
        conn.execute(command)                                                   
      end                                                                       
    end
```

So core test-kitchen has exactly same semantic with ``remote`` word in method...
there are other similar occurrences.
Overall ``remote`` is obvious word for me(as it is opposite to ``local``), in contrast, ``enter_instance`` and ``transport`` are very  misleading.